### PR TITLE
Handle 404 error state in application details view

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-1403

## Description

- Added an error state to handle all application load errors.

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

https://user-images.githubusercontent.com/6041994/181743886-d377ba7a-f72f-4fad-ab90-97f63b967e9a.mov

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by rottencandy — COMMENTED** [review]
What if we used status box? It has custom text for different errors 404, 403, etc. and is used in other places too.
https://github.com/openshift/hac-dev/blob/dcaca376d1bf02ad03a0306ebed303d9bcd4d4e6/src/shared/components/status-box/StatusBox.tsx#L150

File: src/components/ApplicationDetailsView/ApplicationDetailsView.tsx [hunk 1]
```diff
@@ -76,10 +84,24 @@ const ApplicationDetailsView: React.FunctionComponent<ApplicationViewProps> = ({
...
-  if (!applicationsLoaded) {
+  if (!applicationsLoaded && !applicationError) {
     return loading;
   }
 
+  if (applicationError) {
+    return (
+      <Bullseye>
+        <EmptyState variant={EmptyStateVariant.large}>
+          <EmptyStateIcon color="#C9190B" icon={ExclamationCircleIcon} />
```
Comment (sahil143) [inline src/components/ApplicationDetailsView/ApplicationDetailsView.tsx:95]: use color token
```
import { global_palette_red_100 as redColor } from '@patternfly/react-tokens/dist/js/global_palette_red_100';
```

**Review by sahil143 — CHANGES_REQUESTED** [review]

**Comment by rohitkrai03** [comment]
> What if we used status box? It has custom text for different errors 404, 403, etc. and is used in other places too.
> 
> https://github.com/openshift/hac-dev/blob/dcaca376d1bf02ad03a0306ebed303d9bcd4d4e6/src/shared/components/status-box/StatusBox.tsx#L150

@rottencandy I looked at using StatusBox for this but its not updated to look consistent with our current UI in app studio. For some reason the styles were off. Maybe we need to work on a newer StatusBox component for App Studio.

File: src/components/ApplicationDetailsView/ApplicationDetailsView.tsx (hunk 1 above)
Comment (rohitkrai03) [reply src/components/ApplicationDetailsView/ApplicationDetailsView.tsx]: Done.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by rohitkrai03** [comment]
All E2E cypress tests are passing locally on this branch rebased with latest master.
Screenshot 2022-08-04 at 5 48 46 PM

**Pull request merged**